30-remote-site-setup.sh

```diff
@@ -58,7 +65,10 @@ EOF
 root_ssh_dir=/root/.ssh/
 mkdir -p $root_ssh_dir
 chmod 700 $root_ssh_dir
-echo "IdentityFile ${BOSCO_KEY}" > $root_ssh_dir/config
+cat <<EOF > $root_ssh_dir/config
+IdentityFile ${BOSCO_KEY}
+Port $remote_port
```

Comment:
This is at the global scope of .ssh/config - so this Port will apply to _all_ ssh hosts.  Is that really what you want?

Presumably you want Port only to apply to $remote_fqdn, something like

```suggestion
Host $remote_fqdn
  Port $remote_port
```

Reply:
We're only going to SSH to the host specified in `$remote_fqdn` so the global scope is fine. It doesn't hurt to do this, though, I suppose. Could you make the same suggestion for the per-user SSH config so I can bulk commit the suggestions?